### DISCUSSION TITLE
Re-support Ravencoin

### DISCUSSION THREAD

**Discussion by SoraKohaku** [coinfoundry/miningcore#876] [Ideas]
there i try mining and result is cannot mine since it's old one here x16r not support anymore. cause now kawpow support. can implemented future update?

**Comment by oliverw** [comment]
@Konstantin35 Opinions on this?

**Comment by Konstantin35** [comment]
> 
> 
> there i try mining and result is cannot mine since it's old one here x16r not support anymore. cause now kawpow support. can implemented future update?

Yes in the future can be done

**Reply by oliverw** [reply]
Awesome.

**Comment by SoraKohaku** [comment]
Library of kawpow

https://github.com/RavenCommunity/cpp-kawpow/tree/master

**Reply by SoraKohaku** [reply]
Did need used official?They split crypto Lib with algo thing. This make me confused too

**Reply by honorkeles** [reply]
@AlphaX-Projects  did you make it work ?

**Reply by SoraKohaku** [reply]
> @AlphaX-Projects  did you make it work ?

Only have the nodejs so how to remark or convert to kawpow

**Comment by alfirus** [comment]
any update on this matter?

**Reply by oliverw** [reply]
> any update on this matter?

Sorry didn't have the time to work on this as it is quite involved.

**Reply by SoraKohaku** [reply]
Not yet busy

**Reply by SoraKohaku** [reply]
Hard to combine it I still need clue

**Comment by konstiantyn** [comment]
Any updating news?

**Reply by oliverw** [reply]
There's now support for kapow-hashing integrated but the rest of the changes is missing.

**Reply by MiningCryptoLive** [reply]
@oliverw What other changes need to be done? Would like to get involved and get kawpow working for miningcore

**Reply by oliverw** [reply]
> @oliverw What other changes need to be done? Would like to get involved and get kawpow working for miningcore

Well I have already implemented one of the the hardest parts. The libkapow native library. What's missing is managed bindings for it analogous to what's there for libethhash.

**Reply by SoraKohaku** [reply]
> > @oliverw What other changes need to be done? Would like to get involved and get kawpow working for miningcore
> 
> Well I have already implemented one of the the hardest parts. The libkapow native library. What's missing is managed bindings for it analogous to what's there for libethhash. 

Ethash is hard now golang locked. Man hard except they have c releases not golang. How about used old ethash did impact?

**Reply by oliverw** [reply]
> > > @oliverw What other changes need to be done? Would like to get involved and get kawpow working for miningcore
> > 
> > 
> > Well I have already implemented one of the the hardest parts. The libkapow native library. What's missing is managed bindings for it analogous to what's there for libethhash.
> 
> Ethash is hard now golang locked. Man hard except they have c releases not golang. How about used old ethash did impact?

Sorry what??

**Reply by SoraKohaku** [reply]
Cause i see ethhash at kawpow. See they repository i mean on raven repo
